lib/locales/en.ts

```diff
@@ -925,10 +948,12 @@ export const locale: Locale = {
     teresa_of_jesus_of_avila_virgin: 'Saint Teresa of Jesus, Virgin and Doctor of the Church',
     teresa_of_jesus_of_los_andes_virgin: 'Saint Teresa of Jesus of Los Andes, Virgin',
     teresa_of_portugal_religious: 'Blessed Teresa of Portugal, Religious',
+    the_penitent_thief: 'The Penitent Thief',
```

Comment:
No initial definite article in the key names.

Also consider using `Dismas the Good Thief` or `Dismas the Penitent Thief`.

```suggestion
    penitent_thief: 'The Penitent Thief',
```

Reply:
I've renamed the key (and English locale) to `dismas_the_good_thief`, `Dismas the Good Thief`.

Reply:
Oh, I forgot to check that option. 🤦‍♂️ 👍